### PR TITLE
Angular route subgenerator and test added

### PR DESCRIPTION
**Pull request by UndeadBaneGitHub**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,4 +49,4 @@
     "temp": "~0.8.3"
   },
   "license": "MIT"
-}
+}
```
Comment (ilanbiala) [inline package.json]: newline

**Comment by ilanbiala** [comment]
Just the newline and LGTM.

**Comment by UndeadBaneGitHub** [comment]
@ilanbiala @codydaig fixed

**Comment by ilanbiala** [comment]
LGTM.

**Pull request merged**